### ISSUE TITLE
메인 페이지 이미지가 지나치게 큼

### ISSUE THREAD

**Issue by scarf005** [jiphyeonjeon-42/frontend#435]
### 재현 방법

![image](https://github.com/jiphyeonjeon-42/frontend/assets/54838975/451364f4-0993-4a5a-95f2-b3716794804b)

로컬에서 실행후 루트 페이지 (http://localhost:4242) 접속시 이미지 크기가 300x200px로 유지되지 않음

![image](https://github.com/jiphyeonjeon-42/frontend/assets/54838975/6b71520f-c2f4-4ff5-964c-eca4d7dd8808)
px이 아닌 숫자값을 문자열로 넘겨주었기 때문으로 보임

### 발생 이유
[`<img>` 태그의 `width`와 `height`값은 반드시 **정수형**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#width)이어야 하는데 MainNewBook 컴포넌트에서 문자열 타입 숫자 (`"300"`)을 인자로 줌

### 발생 위치

[MainNewBook 컴포넌트](https://github.com/jiphyeonjeon-42/frontend/blob/23e118966e1478935deea51a0eb96f78d2d4b51d/src/component/main/MainNewBook.tsx#L13-L30)
[Image 컴포넌트](https://github.com/jiphyeonjeon-42/frontend/blob/23e118966e1478935deea51a0eb96f78d2d4b51d/src/component/utils/Image.tsx#L24)

### 추가 정보

![image](https://github.com/jiphyeonjeon-42/frontend/assets/54838975/be25fce1-78f6-4d38-ad44-73272d96c0c7)
`"300"` 과 `300`이 동작이 다름

<details><summary>GPT에게 물어본 결과 `"300"`은 올바른 크기가 아님</summary>

# CSS

### You
in css img width, why is `"300"` and `300` different?

### ChatGPT
In CSS, there is a difference between `300` and `"300"` in how they're treated.

If you use `300` without quotation marks, it is interpreted as a length and the default unit is pixels (px). So when you write `img { width: 300; }`, it's the same as `img { width: 300px; }`.

On the other hand, if you use `"300"`, it's a string and not a valid value for the `width` property in CSS. CSS expects a length (like `300px`), a percentage (like `50%`), or `auto` for properties like `width`. If you use a string like `"300"`, CSS won't be able to interpret this as a valid length or percentage, and so it would be ignored, leaving the width unchanged.

In other words, `img { width: "300"; }` would not work correctly because `"300"` is not a valid value for the `width` property in CSS. However, `img { width: 300; }` or `img { width: 300px; }` would be valid.

</details>
